.github/workflows/wheels.yml

```diff
@@ -0,0 +1,276 @@
+name: Build and test wheels
+
+on:
+  push:
+    branches:
+      - master
+      - test
+    tags:
+      - '*'
+  release:
+    types: [published]
+  pull_request:
+
+jobs:
+#  windows:
```

Comment:
I think we should probably delete these blocks for Windows and OSX, or at least put them at the bottom of the file. It'll be confusing for people to come in and see huge chunks of commented out code.

I wouldn't be in any hurry revisiting OSX or Windows binaries, tbh, so probably by the time anyone comes back to this these commented blocks will just be misleading anyway, and (whoever they are) would be better off starting fresh.